src/cluster/cluster-registry.js

```diff
@@ -27,6 +27,7 @@ module.exports = class ClusterRegistry extends EventEmitter {
 	 */
 	constructor( options, connectionEndpoint ) {
 		super();
+		this.setMaxListeners( 20 );
```

Comment:
Was getting a `(node) warning: possible EventEmitter memory leak detected. 11 remove listeners added. Use emitter.setMaxListeners() to increase limit.`